Gearable.hpp

```diff
@@ -1,5 +1,30 @@
 #pragma once
 
+enum class ManualGear : int {
+   reverse = -1,
+   neutral,
+   gear_1st,
```

Comment:
Remove "gear" prefix.

Reply:
I don't think it is a good idea to start a variable name from _

Reply:
Ok.